lib/linux_stats/os/sys_block_stat.rb

```diff
@@ -37,57 +40,65 @@ module Column
   end
 
   module DataFile
-    CPUINFO = '/proc/cpuinfo'
-    DISK_STATS = '/proc/diskstats'
-    SECTOR_SIZE = '/sys/block/sda/queue/hw_sector_size'
+    CPUINFO = '/cpuinfo'
+    DISK_STATS = '/diskstats'
+    SECTOR_SIZE = '/block/sda/queue/hw_sector_size'
   end
 
-  def self.cpuinfo
-    ret = 0
-    IO.readlines(DataFile::CPUINFO).each do |line|
-      ret += 1 if line =~ /^processor/
+  class Reporter
+    def initialize(_data = nil, proc_data_directory = PROC_DIRECTORY_DEFAULT, sys_data_directory = SYS_DIRECTORY_DEFAULT)
```

Comment:
This line is too long to fit on my screen.  Maybe separate each entry w/ a newline?